tools/checklocks/checklocks.go

```diff
@@ -117,6 +122,15 @@ func run(pass *analysis.Pass) (interface{}, error) {
 		pc.exportFunctionFacts(fn)
 	})
 
+	var (
+		pgf   PkgPerfFacts
+		start time.Time
```

Comment:
Move start to the tightest scope it can be declared in. This eliminates interactions.